app/src/features/projects/view/ProjectPage.test.tsx

```diff
@@ -137,37 +143,6 @@ describe('ProjectPage', () => {
     });
   });
 
-  it('sees delete project button as disabled when accessing a published project as a project administrator', async () => {
```

Comment:
question: why was this removed?

Reply:
It was a duplicate test, from what I could tell.  Same test name and same assertions, and same setup

Reply:
Ah I see, ok cool